README.md

```diff
@@ -1,3 +1,7 @@
+<p align="center">
```

Comment:
the generated file looks weird when logo is in the middle :smile: (just my personal opinion)

Reply:
I agree but @qlyoung wanted it centered :grimacing:

Reply:
nah